module/input/Camera/data/config/nugus1/Cameras/Right.yaml

```diff
@@ -24,7 +24,7 @@ lens:
   fov: 180 * pi / 180
   # The homogenous transform from the rigid platform this camera is attached to to its optical location
   Hpc:
-    - [0.9997403694613228, -0.016551656784825577, -0.01566002262642871, 0.08893612063588988]
```

Comment:
revert

Reply:
This will be needed as pitch frame is orientated differently now. I haven't tested on the real robot yet though but was rotation needed in webots. Ideally, we should add a frame in the onshape/urdf model of where the cameras are so we can obtain this matrix dynamically.